Source/EasyNetQ.Management.Client/IManagementClient.cs

```diff
@@ -82,6 +82,31 @@ public interface IManagementClient : IDisposable
     /// <returns></returns>
     Task<IReadOnlyList<Exchange>> GetExchangesAsync(CancellationToken cancellationToken = default);
 
+    /// <summary>
+    ///     A list of all exchanges.
+    /// </summary>
+    /// <param name="pageCriteria"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<PageResult<Exchange>> GetExchangesByPageAsync(PageCriteria pageCriteria, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    ///     A list of all exchanges.
+    /// </summary>
+    /// <param name="vhostName"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<IReadOnlyList<Exchange>> GetExchangesAsync(string vhostName, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    ///     A list of all exchanges.
```

Comment:
the same here

Reply:
https://github.com/EasyNetQ/EasyNetQ.Management.Client/commit/fd3df85311400dd1e4b457a202943bedf92315bf